backend/src/models/user.js

```diff
@@ -11,6 +11,7 @@ const userSchema = new Schema(
         album_ids: { type: [{ address: String, percentage: Number }], default: [] },
         profile_picture: { type: String },
         liked_nfts: { type: [String], default: [] },
+        paid_commisions: {type: [Number]},
```

Comment:
I used a list of numbers instead of a single number since I worried that when users open several windows to create NFTs at the same time, the database might not be updated in the order that NFTs are created and if commissions are not the same all the time, we might fail to remove the correct commission fee.